ariblake/bloom
# Landing page

### Goal
Set up the summer promotion landing page  

### Deployment Instructions
- Deploy to Production theme
- Set settings in the Theme Editor based off of this [theme](https://ariana-fanshawe.myshopify.com/admin/themes/89715572875/editor#/pages/summer-promotion)

### Testing Instructions
- Visit https://edig69ke2m41ne0b-31795118219.shopifypreview.com
  - Ensure that this page matches the wireframe
- Visit https://ariana-fanshawe.myshopify.com/admin/themes/89715572875/editor#/pages/summer-promotion
  - Ensure that the settings are working as intended

### Tested On
FF, Chrome, IE11, EDGE, Android, Safari, iOS

### Code Request Quality Checklist


### Reviewers
@hillary25
